### PR TITLE
Injects data from TSV into metrics table.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/oneoff/populate_metrics_from_tsv [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+end
+
+# Prompts the user for metadata about the data.
+# @return [Hash] The metadata, as a hash.
+def get_metadata
+  print 'COMPUTED ON: '
+  computed_on = gets.chomp
+
+  print 'COMPUTED BY: '
+  computed_by = gets.chomp
```
Comment (Hamms) [inline bin/oneoff/populate_metrics_from_tsv:32]: are these fine as strings, or will they need to be cast to dates somewhere?

File: bin/oneoff/populate_metrics_from_tsv [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# a header row.
+# @return data [Array]
+def get_data
+  print 'TSV FILE: '
+  tsv_file = gets.chomp
+
+  data = []
+  CSV.foreach(tsv_file, headers: true, col_sep: "\t") do |row|
+    data << row
+    puts row
```
Comment (Hamms) [inline bin/oneoff/populate_metrics_from_tsv:20]: how noisy will this be?

File: bin/oneoff/populate_metrics_from_tsv (hunk 1 above)
Comment (ashercodeorg) [reply bin/oneoff/populate_metrics_from_tsv:32]: The actual insert statement uses strings, so casting them to dates (to get casted back to strings) is unnecessary.

File: bin/oneoff/populate_metrics_from_tsv (hunk 2 above)
Comment (ashercodeorg) [reply bin/oneoff/populate_metrics_from_tsv:20]: Noisy enough that I expect to delete it in a future PR. But at least for now, I want the output.

**Pull request merged**